gdpr/models.py

```diff
@@ -25,7 +25,8 @@
 class LegalReasonManager(models.Manager):
 
     def create_consent(self, purpose_slug: str, source_object, issued_at: Optional[datetime] = None,
-                       tag: Optional[str] = None, related_objects: Optional[Iterable[Type[models.Model]]] = None):
+                       tag: Optional[str] = None, related_objects: Optional[Iterable[Type[models.Model]]] = None,
+                       expires_at: Optional[datetime] = None):
```

Comment:
please add expires_at to the Args in docstring

Reply:
@Formulka , `expires_at` mentioned in docstrings